### DISCUSSION TITLE
mspider/user-guide/service-list/service-entry/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1284] [Announcements]
# mspider/user-guide/service-list/service-entry/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/mspider/user-guide/service-list/service-entry/

**Comment by yulng** [comment]


**Reply by liyuerich** [reply]
截图上是 test.service

**Reply by yulng** [reply]
好的

**Comment by liyuerich** [comment]
Location

服务地址。需要输入一个有效的 IP 地址。用于标明服务是否处于网格内部。

应该是 服务位置，也并不需要输入IP地址

**Comment by kebe7jun** [comment]
Endpoint 说明太少了。

**Comment by windsonsea** [comment]
Update some text in PR #5314
